src/actions/run-command.h

```diff
@@ -39,6 +39,8 @@ class RunCommand : public RepeatedAction {
   std::string decreaseCommand;
 
   static bool runCommand(const std::string &command);
+
+  static bool runCommand(const std::string &command, const Gesture& gesture);
```

Comment:
The `&` should go like this:

```suggestion
  static bool runCommand(const std::string &command, const Gesture &gesture);
```